docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.            out = np.r_[out, np.c_[np.full(k, m), top_k_discords]]
</pre><p>This seems like a bad idea. Why not just append to a list and then convert the full list to an array at the very end?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
you mean something like this:
```
out = []
out.append(top_k_discords)
```
And then add the first column that shows the length of subsequences, and return the `numpy.ndarray`?

Reply:
Got it. Will be fixed  in next set of commits.